### ISSUE TITLE
Invalid ChangeSetName

### ISSUE THREAD

**Issue by simpsora** [envato/stack_master#93]
It looks like the code that generates change set names has got a bug:

```
$ stack_master apply us-east-1 awscope
Executing apply on awscope in us-east-1
Stack diff:
...
Parameters diff: No changes
Aws::CloudFormation::Errors::ValidationError 1 validation error detected: Value 'StackMaster2016-04- 1-1014-1459466063' at 'changeSetName' failed to satisfy constraint: Member must satisfy regular expression pattern: [a-zA-Z][-a-zA-Z0-9]*
exit status 1
```

**Comment by simpsora** [comment]
And it's because of:

```
lib/stack_master/change_set.rb:
     def self.generate_change_set_name
      'StackMaster' + Time.now.strftime('%Y-%m-%e-%H%M-%s')
```

I think you want `%d` there (zero-padded day of month) rather than `%e` (space-padded day of month).  PR incoming.

**Comment by stevehodgkiss** [comment]
Fixed in 0.3.1
